src/lib/style/components/_annotations.scss

```diff
@@ -2,108 +2,119 @@
 
 .bootstrap-4-backport {
   .AnnotationEditor {
-    display: grid;
```

Comment:
Diff view makes this change look crazier than it is, just added a couple of classes for the loading spinner and button arrangements